src/Component/Layout.js

```diff
@@ -44,24 +63,33 @@ const Layout = ({modalVisible, setModalVisible, setTodo, todo}) => {
         <View style={styles.centeredView}>
           <View style={styles.modalView}>
             <Text style={styles.modalText}></Text>
-            <Text style={styles.title}>Add Todo</Text>
+            <Text style={styles.title} >Add Todo</Text>
+           
             <TextInput
               style={styles.input}
               onChangeText={setChangeText}
               value={text}
               placeholder=""
+          
             />
             <View style={styles.pop}>
               <Pressable
                 style={[styles.button, styles.buttonClose]}
                 onPress={handleModal}>
                 <Text style={styles.textStyle1}>Cancel</Text>
               </Pressable>
-              <Pressable
+            { 
+              select?
+             <Pressable
                 style={[styles.button, styles.buttonClose]}
-                onPress={handlechange}>
-                <Text style={styles.textStyle2}>Done</Text>
-              </Pressable>
+                onPress={()=>handleEdit(text,id , setChangeText)} >
```

Comment:
`onPress={()=>handleEdit(text,id , setChangeText)} >`
Why are you sending setChangeText to a function?